monash-human-power/MHP-website
# Add MHP Header and Footer

## Description

Enjoy website headers and footers? Well this PR is for you!
+ Footer look finalized
+ Header look finalized
+ Added styled components
+ Body content started (Still in development) 

## Screenshots
Header
![image](https://user-images.githubusercontent.com/23159604/89099371-05dc5300-d432-11ea-9382-8910682060e8.png)

Footer
![image](https://user-images.githubusercontent.com/23159604/89099385-1c82aa00-d432-11ea-8a22-d316350d205b.png)


## Steps to Test (Netlify)
[View here](https://deploy-preview-4--mhp-test.netlify.app/)

## Steps to Test (Locally)

1. Clone the git repo
2. Make sure node and npm is installed
3. Run `npm install -g gatsby-cli` to install Gatsby
4. Run `npm install` to install all dependancies
5. Run `gatsby develop`
